docs/atlascli/command/atlas-dataFederation-privateEndpoints-delete.txt

```diff
@@ -12,7 +12,7 @@ atlas privateEndpoints onlineArchive aws delete
    :depth: 1
    :class: singlecol
 
-Delete a specific Data Lake private endpoint for your project.
+Remove the specified data federation private endpoints from your project.
```

Comment:
Same comment about the singular endpoint here and in the endpointId description on this page. Applies in a few other places as well - I'll call each one out since it may be hard to search for. 

Note: If this *should* be plural, instead clarify in the description that you can specify a list